cmd/protecodeExecuteScan.go

```diff
@@ -336,5 +336,10 @@ func getTarName(config *protecodeExecuteScanOptions) string {
 	}
 	// replace unwanted chars
 	fileName = strings.ReplaceAll(fileName, "/", "_")
+	sha256 := "@sha256"
+	if strings.Contains(fileName, sha256) {
```

Comment:
No need for if, simply replace all of `@sha256:` and if it is not found nothing happens.

Reply:
done as requested